extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/widgets/LogoWidget.kt

```diff
@@ -10,6 +10,8 @@ import com.mapbox.maps.renderer.widget.WidgetPosition
 /**
  * Widget shows compass. Positioned in the bottom left corner by default.
  *
+ * Note: This feature is only available for Android Maps SDK v10.4.0 and above
```

Comment:
💯

Reply:
Could also add a `@since` annotation that [mapbox-java style](https://github.com/mapbox/mapbox-java/blob/f7e78d9f488d2d5ebdfae3ab34021eb24de65b41/services-geojson/src/main/java/com/mapbox/geojson/GeoJson.java#L12)